resources/services/iam/roles.go

```diff
@@ -178,9 +178,14 @@ func resolveIamRoleAssumeRolePolicyDocument(ctx context.Context, meta schema.Cli
 }
 func resolveIamRoleTags(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
 	r := resource.Item.(types.Role)
-	svc := meta.(*client.Client).Services().IAM
+	cl := meta.(*client.Client)
+	svc := cl.Services().IAM
 	response, err := svc.ListRoleTags(ctx, &iam.ListRoleTagsInput{RoleName: r.RoleName})
 	if err != nil {
+		if cl.IsNotFoundError(err) {
+			meta.Logger().Debug("received NoSuchEntity on ListRoleTags, role does not exists", "region", cl.Region, "err", err)
```

Comment:
```suggestion
			meta.Logger().Debug("LstRoleTags: role does not exist", "region", cl.Region, "err", err)
```
Maybe? little bit better, and a generic error message (since IsNotFoundError might check for multiple things)?